modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -380,29 +380,53 @@ a|Write the ISO to a new output file.
 a|`-h`, `--help`
 a|Print help information.
 
-2+|*coreos-installer PXE Ignition subcommands*
+2+|*coreos-installer PXE subcommands*
 
 |*_Subcommand_* |*_Description_*
 
 2+|Note that not all of these options are accepted by all subcommands.
 
+a|`coreos-installer pxe customize <options> <path`
+a|Customize a {op-system} live PXE boot config.
+
 a|`coreos-installer pxe ignition wrap <options>`
 a|Wrap an Ignition config in an image.
 
 a|`coreos-installer pxe ignition unwrap <options> <image_name>`
 a|Show the wrapped Ignition config in an image.
 
-2+|*coreos-installer PXE Ignition subcommand options*
+2+|*coreos-installer PXE customize subcommand options*
 
 |*_Option_* |*_Description_*
 
 2+|Note that not all of these options are accepted by all subcommands.
 
-a|`-i`, `--ignition-file <path>`
-a|The Ignition config to be used. Default is `stdin`.
+a|`--dest-ignition <path>`
+a|Merge the specified Ignition config into a new configuration fragment for the destination system.
+
+a|`--dest-device <path>`
+a|Install and overwrite the specified destination device.
+
+a|`--network-keyfile <path>`
+a|Configure networking by using the specified NetworkManager keyfile for live and destination systems.
+
+a|`--ignition-ca <path>`
+a|Specify an additional TLS certificate authority to be trusted by Ignition.
+
+a|`--pre-install <path>`
+a|Run the specified script before installation.
+
+a|`post-install <path>`
+a|Run the specified script after installation.
+
+a|`--installer-config <path>`
+a|Apply the specified installer configuration file.
+
+a|`--live-ignition <path>`
+a|Merge the specified Ignition config into a new configuration fragment for the live environment.
 
 a|`-o,` `--output <path>`
-a|Write the ISO to a new output file.
+a|Write the initramfs to a new output file.
```

Comment:
Thanks, I've added a small note. But since it's required maybe it would be worth adding to the subcommand description:
```
coreos-installer pxe customize <options> -o <path>
```
or something similar 🤔

Reply:
Works for me!